contracts/settling_game/L03_Buildings.cairo

```diff
@@ -464,27 +613,52 @@ func fetch_buildings_by_type{
 
     # state contract
     let (buildings_state_address) = IModuleController.get_module_address(
-        contract_address=controller, module_id=6)
+        contract_address=controller, module_id=ModuleIds.S03_Buildings)
 
     let (local data) = IS03_Buildings.get_realm_buildings(buildings_state_address, token_id)
 
-    let (local castles) = unpack_data(data, 0, 255)
-    let (local markets) = unpack_data(data, 8, 255)
-    let (local aquaducts) = unpack_data(data, 16, 255)
-    let (local ports) = unpack_data(data, 24, 255)
-    let (local barracks) = unpack_data(data, 32, 255)
-    let (local farms) = unpack_data(data, 40, 255)
-    let (local temples) = unpack_data(data, 48, 255)
-    let (local shipyards) = unpack_data(data, 56, 255)
+    let (local Fairgrounds) = unpack_data(data, 0, 63)
```

Comment:
No need to change anything, just commenting for knowledge sharing (I learnt this recently and it surprised me). Since Cairo 0.6.0, there's no need to declare `local` in `let (local foo)` with `alloc_locals`, the compiler can supposedly figure it out by itself now 🎉

Reply:
nice! ok I will clean up, no point it being there